src/micro_http/src/request.rs

```diff
@@ -8,6 +8,9 @@ pub use common::RequestError;
 use common::{Body, Method, Version};
 use headers::Headers;
 
+// This type represents the RequestLine raw parts.
```

Comment:
nit: A comment with what the slices represents would be nice and further improve the readability of the code.